.env.example

```diff
@@ -1,4 +1,6 @@
 ADO_PAT=(PAT from Azure DevOps)
 ADO_URL=(https://dev.azure.com/ORG)
+APPLICATIONINSIGHTS_CONNECTION_STRING=(Connection string from Azure Application Insights, or blank if not using)
 ASANA_TOKEN=(PAT from Asana)
 ASANA_WORKSPACE_NAME=(Name of the Asana Workspace)
+OTEL_SERVICE_NAME=sync
```

Comment:
**Suggestion:** Consider adding comments to the environment variables in the `.env.example` file. This would make the file easier to understand for new developers.
```suggestion
# Suggestion: Add comments to the environment variables in the `.env.example` file
```